panel/browser-panel.cpp

```diff
@@ -16,6 +16,10 @@
 #include <util/base.h>
 #include <thread>
 
+#if !defined(_WIN32) && !define(__APPLE__)
```

Comment:
Shouldn't this be `#if !defined(_WIN32) && !defined(__APPLE__)`?
Note that you use `define` instead of `defined` for the second condition.